genanki/package.py

```diff
@@ -20,6 +20,7 @@ def __init__(self, deck_or_decks=None, media_files=None):
       self.decks = deck_or_decks
 
     self.media_files = media_files or []
+    self.media_files = list(set(media_files))
```

Comment:
You need to handle the case where `media_files = None`. `list(set(None))` raises an exception.